src/App.js

```diff
@@ -0,0 +1,85 @@
+import React, { useState, useEffect } from 'react';
+import {
+    BrowserRouter as Router,
+    Switch,
+    Route,
+    Link
+  } from "react-router-dom";
+
+import db from './db';
+import AddItem from './AddItem';
+import ItemList from './ItemList';
+
+const tableName = 'applications';
+
+const initState = {
+    applications: []
+};
+function App() {
+    const [state, setState] = useState(initState);
+    useEffect(() => {
+        db.table(tableName)
+            .toArray()
+            .then((applications) => {
+                console.log('got applications from db: ', applications);
+                setState({ applications });
+            });
+    }, []);
```

Comment:
nit: eslint-disable-next-line removes warning for empty dependency array

Reply:
No warning showing up here, will leave as is.